### ISSUE TITLE
data generator random seed specification not working?

### ISSUE THREAD

**Issue by mattroos** [MichalBusta/E2E-MLT#14]
I'd like to be able to train in a repeatable manner--in particular, generate batches of images such that every time I start training from scratch, the images are presented for training in the same order.  the data_util.GeneratorEnqueuer class accepts a random_seed parameter, but even when I set it a fixed number, the images are in a different order each time I train.  I'm only using one reader. @MichalBusta, any ideas?

**Comment by MichalBusta** [comment]
Hi Matt, 
data_gen.py, line 574, 590  

probably best way to do it: 
 - shuffle train list offline 
 - disable randomness (data_gen.py, line 574, 590) 

best way will be reuse 'shuffle' parameter from generic dataloader.

**Comment by mattroos** [comment]
Thanks, that's what I was looking for.  Although for others' awareness, the transforms will still have randomness.  The code lines below may be enough to make everything deterministic.  But see the link for more discussion.
```
torch.manual_seed(0)
torch.cuda.manual_seed(0)
np.random.seed(0)
random.seed(0)
```
https://github.com/pytorch/pytorch/issues/7068
